ui.qml.in

```diff
@@ -867,6 +867,13 @@ Item {
             [8, "Low Battery"],
             [9, "Board Idle"],
             [10, "Other"],
+            [11, "Cell Under Temp"],
+            [12, "Cell Over Temp"],
+            [13, "Cell Low Voltage"],
+            [14, "Cell High Voltage"],
+            [15, "Cell Balance"],
+            [16, "BMS Connection"],
+            [17, "BMS Temp"],
```

Comment:
I wonder if all these should be prefixed with "BMS: " just to make it clear where it comes from. For the last two items specifically:
> BMS: Connection Lost
BMS: BMS Temp

 (yeah, the double BMS is not very pretty on the last one but I don't have a better idea)

Reply:
The beep reason is already quite long as far as display in UI and I don't think there's ambiguity as to what Cell means.

Reply:
I disagree, I think for casual users clearly stating it's an error coming from BMS is important and helpful. There's room in the UI. I'd like all the warnings coming from the BMS to be clearly differentiated.

Reply:
ok